src/PaymentTest.java

```diff
@@ -0,0 +1,8 @@
+public class PaymentTest {
+
+    public static void main(String[] args) {
+        System.out.println("Payment test1");
+        System.out.println("Payment test3");
+        System.out.println("Payment test4");
```

Comment:
Looks like you have to add some more code

Reply:
no thank you